arduino/discovery/discovery.go

```diff
@@ -206,7 +206,7 @@ func (disc *PluggableDiscovery) jsonDecodeLoop(in io.Reader, outChan chan<- *dis
 			closeAndReportError(err)
 			return
 		}
-		logrus.Infof("from discovery %s received message %s", disc.id, msg)
+		logrus.Infof(tr("from discovery %s received message %s"), disc.id, msg)
```

Comment:
```suggestion
		logrus.Infof(tr("from discovery %[1]s received message %[2]s"), disc.id, msg)
```

As mentioned above, I am opposed to the arbitrary internationalization of log messages. However, since you are interested in assisting in the internationalization effort I will add a note about how we handle this type of string:

[Indices](https://pkg.go.dev/fmt#:~:text=explicit%20argument%20indexes) should be added to `printf` verbs when multiple are present in a string.

This allows the translator to easily insert them at the appropriate location in the sentence structure without being required to understand [the Go printf syntax](https://pkg.go.dev/fmt#:~:text=explicit%20argument%20indexes) and without being restricted to their relative order in the English language source string.